.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+[*, *.json, *.js]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.md]
```

Comment:
I wouldn't impose rules on markdown files because we use them to make tests, and sometimes we want to break out of all these rules. Is there a way to apply this only to documentation files?

Reply:
I agree. This could cause problems if it is not limited the docs.
Maybe remove this check for *.md and it can be added when we create a `/docs` folder with GitHub Pages.